src/package.test.ts

```diff
@@ -15,12 +16,14 @@ import {
   readMonorepoRootPackage,
   readMonorepoWorkspacePackage,
   updatePackage,
+  updatePackageChangelog,
 } from './package';
 import * as fsModule from './fs';
 import * as packageManifestModule from './package-manifest';
 import * as repoModule from './repo';
 
-jest.mock('@metamask/auto-changelog');
+const outdent = _outdent({ trimTrailingNewline: false });
```

Comment:
Nice that a package can do this for us. Heads up that we already have a version of this in `tests/functional/helpers/utils.ts`, called `normalizeMultilineString` — we might want to replace that with `outdent` at some point.